core/base/ftmTree/FTMTreeUtils.h

```diff
@@ -0,0 +1,295 @@
+/// \ingroup base
+/// \class FTMTreeUtils
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Utils function for manipulating FTMTree class
+
+#ifndef _FTMTREEUTILS_H
+#define _FTMTREEUTILS_H
+
+#pragma once
+
+#include <FTMTree_MT.h>
+#include <stack>
+
+using namespace ttk;
+using namespace ftm;
+
+// --------------------
+// Utils
+// --------------------
+
+void myPause();
+
+void printTreesStats(std::vector<ftm::FTMTree_MT *> &trees);
```

Comment:
Please declare these functions in the `ftm` namespace.
(for all the functions of this header)

Reply:
Ok! Done in the next commit.